src/hooks/v2/NftRewards.ts

```diff
@@ -19,10 +21,373 @@ export default function useNftRewards(
       return
     }
 
-    const getRewardTiers = async () => {
+    const getRewardTiers = () => {
       return Promise.all(CIDs.map(cid => getRewardTierOfCid(cid)))
     }
 
     return getRewardTiers()
   })
 }
+
+export const TEMPORARY_NFT_DEPLOYER_ABI: ContractInterface = [
```

Comment:
**suggestion** suggest moving this to own file under `contracts/rinkeby`

Reply:
Yup